src/components/game/GameCard.js

```diff
@@ -40,6 +40,14 @@ const RadioButton = ({ label, value, onChange }) => {
 const CardBody = ({ article, nextButton }) => {
   const [selection, setSelection] = useState("");
   const [isGuessCorrect, setIsGuessCorrect] = useState("");
+  const [score, setScore] = useState(() => {
+    const savedScore = localStorage.getItem("score");
+    return savedScore ? JSON.parse(savedScore) : 0;
+  });
```

Comment:
I love this refactor. So, we are defining our variables, calling useState, passing in the anonymous function that (because it's an or statement), first checks for a localStorage score to display, and if there isn't one, it display 0. Cleaned up!